src/Common/MoistThermodynamics/isentropic.jl

```diff
@@ -0,0 +1,44 @@
+export IsentropicProcess
```

Comment:
```suggestion
export DryAdiabaticProcess
```
(and similar throughout). I'd make clear that these are all dry isentropic (or adiabatic) processes.